### PR TITLE
[for 2.6.x] Fix/2764 TinyMCE height responsivity

### PR DESCRIPTION
**Pull request by metzgegu**
#2764 

## Checkpoints

The goal of this section is to help code reviewer and merge request author to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewer**

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developper**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [x] Tested by quality team

### CODE REVIEW

File: frontend_lib/src/component/Timeline/Timeline.jsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ class Timeline extends React.Component {
 
         {props.loggedUser.userRoleIdInWorkspace >= ROLE.contributor.id &&
           <form className={classnames(`${props.customClass}__texteditor`, 'timeline__texteditor')}>
-            <div className={classnames(`${props.customClass}__texteditor__textinput`, 'timeline__texteditor__textinput')}>
+            <div className={classnames(`${props.customClass}__texteditor__textinput`, 'timeline__texteditor__textinput', props.wysiwyg ? 'timeline__texteditor__textinput__wysiwyg' : null)}>
```
Comment (Skylsmoi) [inline frontend_lib/src/component/Timeline/Timeline.jsx]: line is too long, add line break for each parameters of classnames()

File: frontend/src/css/Generic.styl [hunk 2]
```diff
@@ -42,6 +42,22 @@ div.mce-fullscreen
   z-index 1050
   &.mce-container
     top 61px
+/*Editing container*/
```
Comment (Skylsmoi) [inline frontend/src/css/Generic.styl]: add a comment with a link to the stackoverflow answer you used for this.

File: frontend/src/css/Generic.styl [hunk 3]
```diff
@@ -42,6 +42,22 @@ div.mce-fullscreen
...
+.mce-tinymce, .mce-container-body
+  min-height 100%
+/*Editing area*/
+.mce-container-body .mce-edit-area
+  position absolute
```
Comment (Skylsmoi) [inline frontend/src/css/Generic.styl]: I'm not a big fan of absolute solution.
The stackoverflow answer you linked in the issue shows another solution using flexbox.
The flexbox solution is cleaner and more recent than the accepted one.
If you didn't try it, do it and if it works, prefer this one

Comment (metzgegu) [reply frontend/src/css/Generic.styl]: I already try it and re-try it but it didn't work well and I didn't find a way to fix it.

Comment (Skylsmoi) [reply frontend/src/css/Generic.styl]: ok then

**Comment by Skylsmoi** [comment]
missing tests or info about not having tests

**Comment by metzgegu** [comment]
This issue is a responsive only issue and I don't think we can have good test which will be relevant enough and stable enough (in Cypress)

**Comment by Skylsmoi** [comment]
@metzgegu This is just a personal opinion.
I don't disagree with you but you need to explain why

**Comment by gbonaspetti** [comment]
Tested and verified on branch.

**Comment by metzgegu** [comment]
Cypress test here would not be relevant enough because with cypress we could test if the height of tinyMCE input has changed but it will not test if it has the exact correct height according to the new viewport.

**Pull request merged**